cypress/e2e/alerts.cy.js

```diff
@@ -1,25 +1,60 @@
-describe('Cypress application', () => {
-  before(() => {
+/// <reference types='cypress' />
+const faker = require('faker');
 
+describe('Cypress application', () => {
+  beforeEach(() => {
+    cy.visit('/');
   });
 
-  it('should have the ability to assert automatically resolved alerts', () => {
+  it('should be able to assert alerts that are resolved automatically', () => {
+    cy.get('#alertButton').click();
 
+    cy.on('window:alert', (alertText) => {
+      expect(alertText).to.equal('You clicked a button');
+    });
   });
 
-  it('should have the ability to assert scheduled allert', () => {
+  it('should be able to assert alerts that are scheduled', () => {
+    cy.get('#timerAlertButton').click();
+    cy.clock();
+    cy.tick(5000);
 
+    cy.on('window:alert', (alertText) => {
+      expect(alertText).to.equal('This alert appeared after 5 seconds');
+    });
   });
```

Comment:
This test case could be more descriptive as well. For example, 'should be able to assert alerts that are scheduled 5 seconds after button click'.